Add release notes for upcoming Bio-Formats 8.3.0 release (#432)
ome/bio-formats-documentation: sphinx/about/whats-new.rst

Version history
===============

8.3.0 (2025 August)
-------------------

New file formats:

* Zeiss XRM:
   - added support for .txm and .txrm files
	 
File format fixes:

* LiFlim
   - cast integers to double for division accuracy
* Metamorph
   - don't reset dimensions if the well count and series count match
   - fixed the encoding used for reading the .nd file
* Metamorph TIFF
   - threw more explicit exceptions if parsing invalid TIFF
* Nikon ND2
   - added TextInfoItems to original metadata before parsing
   - changed the padding for a specific condition to pad to 4 byte boundaries.

> include (thanks to Matthew Smith)

Reviewer: ```suggestion
(thanks to Matthew Smith)
```

Reviewer: I think lines 23/24 need to be combined to fix the build.

Author: Should be fixed in [85c7ff3](https://github.com/ome/bio-formats-documentation/pull/432/commits/85c7ff31d915075f1a2075d6437952491fa10d6b)

   - removed the usage of position metadata when distinguishing between T and series
* OME-TIFF
   - fixed optimal tile size if an IFD is not linked to an Image
* Operetta/Harmony
   - fixed case where index XML is in the parent of the image directory
* SlideBook 7
   - fixed miscellaneous errors (thanks to Nicola Papp)
* Tecan Spark Cyto Workspace
   - used try-with-resources when opening SQL connection
* Ventana BIF
   - fixed LEFT overlap and added Z stack support
* Zeiss CZI

>    - fixed LighSource population to include IDs and link LightSources to Channels

Reviewer: ```suggestion
   - fixed LightSource population to include IDs, link LightSources to Channels, and set Attenuation
```

   - fixed IDs and reference issues for Dichroic and Detector objects
   - used millisecond units for exposure times attached to channels

Bio-Formats improvements:

> * showinf: configure logging for -omexml-only after all arguments parsed

Reviewer: ```suggestion
* showinf: configure logging for `-omexml-only` after all arguments parsed
```
